### ISSUE TITLE
Using isomeric smiles with deepchem

### ISSUE THREAD

**Issue by robbiebarrat** [deepchem/deepchem#1493]
I'm trying to train a neural network to discern between active and inactive molecules from a pubchem assay. We have been using canonical smiles to train, but we've found that there are molecules with the same canonical smiles, but distinct isometric smiles and distinct activity levels.

Is there a way to use isometric smiles as an input with deepchem? Or at least a way to encode the information not present in canonical smiles?

**Comment by tomwenseleers** [comment]
I agree this would be cool and useful for many applications... Note that there is also canonical isomeric smiles; this may potentially be useful:
https://github.com/openforcefield/cmiles
